js/src/controls/SplitMapControl.js

```diff
@@ -27,23 +27,25 @@ var LeafletSplitMapControlView = LeafletControlView.extend({
     },
 
     create_obj: function () {
-        var left_model = this.model.get('left_layer');
-        var right_model = this.model.get('right_layer');
+        var left_models = this.model.get('left_layer');
+        var right_models = this.model.get('right_layer');
         var layersModel = this.map_view.model.get('layers');
-        layersModel.push(left_model, right_model);
+        left_models.forEach((left_model)=>{layersModel.push(left_model)});
```

Comment:
You could use the concat method: `layersModel = layersModel.concat(left_models);`